libdnf.spec

```diff
@@ -35,6 +35,7 @@ BuildRequires:  gcc
 BuildRequires:  libsolv-devel >= %{libsolv_version}
 BuildRequires:  pkgconfig(librepo)
 BuildRequires:  pkgconfig(check)
+BuildRequires:  redhat-rpm-config
```

Comment:
Actually, it looks like it's a bug in both `gobject-introspection` and `gcc` packages.